hex/Hex2.resx

```diff
@@ -0,0 +1,120 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Sorry, this was generated by Visual Studio. I'm still learning Github and haven't got down what files to remove from commits yet. I tried to remove all the unnecessary changes from the commit but I must have missed some. This change can be removed without affecting the program.